doc/sphinx/source/recipes/recipe_perfmetrics.rst

```diff
@@ -11,104 +11,118 @@ The goal is to create a standard recipe for the calculation of performance metri
 -----------------------------------
 
-Recipes are stored in nml/
+Recipes are stored in recipes/
 
-* recipe_perfmetrics_CMIP5.xml
+* recipe_perfmetrics_CMIP5.yml
 
-Diagnostics are stored in diag_scripts/
+Diagnostics are stored in diag_scripts/perfmetrics/
 
-* perfmetrics_grading.ncl: calculates grades according to a given metric, with different options for normalization. It requires fields precalculated by perfmetrics_main.ncl.
-* perfmetrics_grading_collect.ncl: collects results from metrics previously calculated by perfmetrics_grading.ncl and passes them to the plotting functions.
-* perfmetrics_main.ncl: calculates and (optionally) plots annual/seasonal cycles, zonal means, lat-lon fields and time-lat-lon fields from input monthly 2-d or 3-d ("T2M", "T3Ms") data. The calculated fields can be also plotted as difference w.r.t. a given reference model. They are also used as input to calculate grading metrics (see perfmetrics_grading.ncl).
-* perfmetrics_taylor.ncl: calculates grades according to a given metric, with different options for normalization. It requires fields precalculated by perfmetrics_main.ncl.
-* perfmetrics_taylor_collect.ncl: collects results from metrics previously calculated by perfmetrics_taylor.ncl and passes them to the plotting functions.
+* main.ncl: calculates and (optionally) plots annual/seasonal cycles, zonal means, lat-lon fields and time-lat-lon fields. The calculated fields can also be plotted as difference w.r.t. a given reference dataset. main.ncl also calculates RMSD, bias and taylor metrics. Input data have to be regridded to a common grid in the preprocessor. Each plot type is created by a separated routine, as detailed below.
+* cycle.ncl: creates an annual/seasonal cycle plot.
+* zonal.ncl: creates a zonal (lat-pressure) plot.
+* latlon.ncl: creates a lat-lon plot.
+* cycle_latlon.ncl: precalculates the metrics for a time-lat-lon field, with different options for normalization.
+* collect.ncl: collects and plots the metrics previously calculated by cycle_latlon.ncl.
 
 User settings
 -------------
 
-User setting files (cfg files) are stored in nml/cfg_perfmetrics/CMIP5/
-
-#. perfmetrics_grading.ncl
-
-   *diag_script_info attributes*
-
-   * MultiModelMean: calculate multi-model mean (True, False)
-   * MultiModelMedian: calculate multi-model median (True, False)
-   * metric: applied metric ("RMSD" = root-mean square difference, "BIAS" = mean bias, "stddev_ratio" = ratio of standard deviations of var and ref (for Taylor diagrams only), "correlation" = pattern correlation of var and ref (for Taylor diagrams only)).
-   * normalization: applied normalization ("mean" = normalization with mean, "median" = normalization with media, "centered_median" = substracting and dividing by the median, "stddev_mean" = normalization with substracting the mean and dividing by the standard deviation)
-
-#. perfmetrics_grading_collect.ncl
+#. main.ncl
 
    *Required diag_script_info attributes*
```

Comment:
It would be better to tell users how to specify this in the recipe instead of in NCL code

Reply:
I'm not sure I understand your point: those listed here are the recipe settings which are expected by that script.

Reply:
Ok, I will use it as a template.
I guess the documentation of other NCL-based recipes needs to be revised accordingly (@axel-lauer @hb326).